### ISSUE TITLE
Not working player.give_named_item

### ISSUE THREAD

**Issue by alsmir** [Source-Python-Dev-Team/Source.Python#72]
Please check the next code:
```python
from events import Event
from players.entity import PlayerEntity
from players.helpers import index_from_userid

@Event('player_activate')
def player_activate(game_event):
    player = PlayerEntity(index_from_userid(game_event.get_int('userid')))
    player.give_named_item('weapon_shotgun', 0, None, False)
```

This code should give the player chosen weapon. But now it's not working.

env:
sourcepython: last edition from 23 september 2015
os: Ubuntu
game: CSGO

**Comment by alsmir** [comment]
env:
sourcepython: last edition from 23 september 2015
os: Ubuntu
game: CSGO

**Comment by satoon101** [comment]
Have you tried using an item name that actually exists in CS:GO?  There is no item weapon_shotgun.  There are 4 shotgun items: weapon_nova, weapon_xm1040, weapon_sawedoff, and weapon_mag7.

**Comment by alsmir** [comment]
I tried but any from these weapons is not working.

**Comment by satoon101** [comment]
Are you certain your server is up to date?  We updated the offsets after the last update, so they should be correct unless there was another update today I don't know about.  Also, make sure that you extracted all the files from the most recent build and overwrite the old files.

**Comment by high-voltage** [comment]
Offsets need to be updated. I found the new ones for:

```[[drop_weapon]]
        offset_linux = 284
        offset_windows = 283
        arguments = POINTER, POINTER, POINTER
        return_type = VOID

[[give_named_item]]
        offset_linux = 449
        offset_windows = 448
        arguments = STRING, INT, POINTER, BOOL
        return_type = POINTER```

**Comment by satoon101** [comment]
My fault, I thought we updated CS:GO, but we only updated OB/CS:S.  I will get those fixed.  Thank you for reporting.

**Comment by satoon101** [comment]
Fixed in 6a46f3da522ffe1f226dc9ed6607d15af22b306d (forgot to add to commit message...)
